examples/question_answering/training_example.py

```diff
@@ -0,0 +1,45 @@
+from datasets import load_dataset
+import csv
+from happytransformer.happy_question_answering import  HappyQuestionAnswering
+
+
+def main():
+    # Be careful not to commit the csv files to the rep
```

Comment:
perhaps they should be added to the gitignore?

Reply:
good idea